app/scripts/modules/core/src/widgets/spinners/Spinner.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+
+const Logo = require('./logo.svg');
+
+export interface ISpinnerProps {
+  size?: string;
+  message?: string;
+  postnote?: string;
+}
+
+export class Spinner extends React.Component<ISpinnerProps, {}> {
+  public static propTypes = {
+    size: React.PropTypes.oneOf(['nano', 'small', 'medium', 'large', 'page']),
+    message: React.PropTypes.string,
+    postnote: React.PropTypes.string
+  }
```

Comment:
Not sure where we landed on `PropTypes` yesterday but thought we agreed they weren't necessary, since we're using Typescript.

I'm not familiar enough with React to know if that's really where we landed.